ReactAndroid/src/main/java/com/facebook/react/views/imagehelper/ResourceDrawableIdHelper.java

```diff
@@ -66,12 +71,40 @@ public int getResourceDrawableId(Context context, @Nullable String name) {
     }
   }
 
+  private String sanitizeResourceDrawableId(@NonNull String name) {
+    return name.toLowerCase().replace("-", "_");
```

Comment:
Can we avoid this by updating our existing usage of "ReactAssetUtils" or w/e to just use the "_" format?  We don't use it that much and it's one less special thing to maintain in the fork.